src/visualizers/VisualizerCycles.ts

```diff
@@ -0,0 +1,270 @@
+        for (let i = 0; i < this.workingSequence.length; i++) {
+            if (this.workingSequence[i] > maxCycle) {
+                maxCycle = this.workingSequence[i]
+            }
+        }
+
+        for (let i = 0; i < this.workingSequence.length; i++) {
+            const x = (800 / this.width) * (i % this.width)
+            const y = (800 / this.width) * Math.floor(i / this.width)
+            this.sketch.fill(100, 100, 0)
+
+            this.sketch.fill(
+                (this.workingSequence[i] * 255) / maxCycle,
+                100,
+                100
+            )
+
+            //Draw each of the squares
+            this.sketch.noStroke()
+            this.sketch.square(x, y, 800 / this.width)
+
+            //Show black lines around each box
+            if (this.displayBoxLines) {
+                this.sketch.stroke(1, 1, 0)
+                this.sketch.square(x, y, 800 / this.width)
+            }
+            this.sketch.noStroke()
+        }
+    }
+
+    // draw() contains all of the mouseOver functions
+    draw() {
+        super.draw()
+
+        this.sketch.colorMode(this.sketch.HSB)
+        this.sketch.textFont('Helvetica')
+
+        //x and y-coordinate of mouse
+        const x = this.sketch.mouseX
+        const y = this.sketch.mouseY
+
+        // find the y-coordinate to set the mouseOver info
+        const tempY =
+            (800 / this.width) * Math.floor(this.numberOfTerms / this.width)
+        let lastY = 0
+        if (tempY < 720) {
+            lastY = tempY
+        }
+        if (tempY >= 720) {
+            lastY = 720
+        }
+        const squareWidth = 800 / this.width
+        const indexOfSquareMouseOver =
+            Math.floor(x / squareWidth)
+            + this.width * Math.floor(y / squareWidth)
+
+        //background large rectangle
+        this.sketch.fill(225, 50, 100)
+        this.sketch.rect(0, lastY, 800, 80)
+
+        // text settings for top of mouseOver info
+        this.sketch.fill(0, 0, 100)
+        this.sketch.textSize(25)
+
+        if (indexOfSquareMouseOver < this.workingSequence.length) {
+            let m = this.seq.getElement(
+                indexOfSquareMouseOver + this.seq.first
+            )
+            console.log(indexOfSquareMouseOver, m)
+
+            const mainText =
+                `Starting value: ${m} Cycle value: `
+                + `${collatzStepsTo1(m)} Sequence: `
+            this.sketch.text(mainText, 15, lastY + 27)
+
+            //smaller background rect for sequence
+            this.sketch.fill(225, 75, 100)
+            this.sketch.rect(18, lastY + 40, 765, 30, 10)
+
+            // text settings for the sequence list
+            this.sketch.textStyle('bold')
+            this.sketch.textSize(15)
+            this.sketch.fill(0, 0, 0)
+            this.sketch.text(m.toString(), 25, lastY + 60)
+
+            if (this.sketch.mouseIsPressed) {
+                // white outline on the square that the mouse is over
```

Comment:
Feel free to address this comment by simply filing an issue for later:  but it might be nice to be able to "unclick" a box also, to forget it.  I tried that but it didn't work.

Reply:
I agree both with the sentiment that clicking should toggle the display of the white border and that it doesn't need to be handled within this initial PR.